src/explore-education-statistics-frontend/src/modules/data-catalogue/DataSetFilePage.tsx

```diff
@@ -271,11 +276,29 @@ export default function DataSetFilePage({
                   />
 
                   {apiDataSetVersionChanges && (
-                    <DataSetFileApiChangelog
-                      changes={apiDataSetVersionChanges}
-                      guidanceNotes={apiDataSetVersion.notes}
-                      version={apiDataSetVersion.version}
-                    />
+                    <>
+                      <DataSetFileApiChangelog
+                        changes={apiDataSetVersionChanges}
+                        guidanceNotes={apiDataSetVersion.notes}
+                        version={apiDataSetVersion.version}
+                      />
+                      {apiDataSetVersionChanges.patchHistory &&
+                        apiDataSetVersionChanges.patchHistory.map(
+                          (change: ApiDataSetVersionChanges) => (
+                            <DataSetFileApiChangelog
+                              key={change.versionNumber.patch}
+                              renderHeading={false}
+                              changes={change}
+                              guidanceNotes={change.notes}
+                              version={
+                                change.versionNumber.patch > 0
+                                  ? `${change.versionNumber.major}.${change.versionNumber.minor}.${change.versionNumber.patch}`
+                                  : `${change.versionNumber.major}.${change.versionNumber.minor}`
+                              }
+                            />
+                          ),
+                        )}
```

Comment:
Using `DataSetFileApiChangelog` for this renders it in a `section` element without a heading, semantically sections should (almost) always have headings as they denote a defined section of content. In this case I think it makes sense for the patch history to be within the API Changelog section, so I suggest passing the `patchHistory` into `DataSetFileApiChangelog` instead and then rendering it there. If you did this you wouldn't need the changes to `DataSetFilePageSection`